RussianFilter/sections/css_extended.txt

```diff
@@ -229,6 +229,8 @@ footballhd-news.com##div[id="dle-content"] > div.lastnewscat:has(> div[id*="Scri
 footballhd-news.com##div[id="dle-content"] > div.lastnewscat:has(> .adsbygoogle)
 !
 lvnews.org.ua#?#.container-margin div[id]:contains(загрузка...) 
+openssource.biz#?##sidebar > div.widget:has(> div.textwidget a[target="_blank"]:not([href*="openssource.biz/"]):not([href="https://openssource.net"]):not([href="https://openssource.pro"]))
```

Comment:
`:not([href^="https://openssource."])`
also this rule will block all links, which does not contain site address

Reply:
look:
![image](https://user-images.githubusercontent.com/8361299/63366803-2e834e00-c383-11e9-9c2a-86134e13d2a7.png)